packages/unified-pipeline/src/utils/ParserUtils.tsx

```diff
@@ -148,3 +149,158 @@ const getStepNode = (step: Record<string, any>, stepIndex: number): Node => {
 const getPlaceholderIcon = (stepIndex: number): React.ReactElement => {
   return stepIndex % 2 ? <Bitbucket /> : <Slack />
 }
+
+/* Parser Utils for V0 pipeline yaml */
+
+const V0_STEPS_PATH_PREFIX = 'stage.spec.execution.steps'
+
+export const parseV0PipelineYaml = ({
```

Comment:
Do you see any potentially repeating/duplicated code between this and V1 yaml parser? Not needed right now, but we should try to move out the common pieces.